### DISCUSSION TITLE
Post processing

### DISCUSSION THREAD

**Discussion by wWL2771140158** [xcompact3d/Incompact3d#229] [Q&A] [answered]
Hello, after running a three-dimensional cylinder flow case, the large vortex structure I obtained is composed of many scattered and fragmented small vortices. How can I obtain a more complete and easily recognizable vortex structure? Meanwhile, my calculation results are three-dimensional. Can I obtain a schematic diagram of the vortex structure on a two-dimensional plane from this?

If anyone could provide any suggestions, I would greatly appreciate it.

**Comment by slaizet** [comment]
Hi,
The Reynolds number for the 3D cylinder case is 300 so you get a fully turbulent flow field. If you want a steady solution you will need to reduce the Reynolds number
Best,
Sylvain

**Reply by wWL2771140158** [reply]
Hello, thank you for your reply. If I choose a higher Reynolds number, can I obtain a clear vortex structure diagram similar to the one shown in the following figure in the post-processing? Is this related to the identification criteria of vortices?
![image](https://github.com/xcompact3d/Incompact3d/assets/103632822/9fb4a9c6-6e76-40ae-8a7d-242beb283c60)

**Reply by slaizet** [reply]
If you put two cylinders in the simulation and if you use the same set up as in the reference then you will get a similar flow field.
Best,
Sylvain

**Reply by wWL2771140158** [reply]
Excuse me again, what I mean is that when I used Paraview for post-processing, the obtained vorticity diagram is as shown in the figure. At this time, there is no obvious alternating vortex band in the vorticity diagram, and the upper and lower vortex bands cannot be clearly identified. I would like to obtain a clearer and more intuitive vortex structure from the reference article in the previous reply. Are there any tips to pay attention to in post-processing? Is there any other post-processing software that can achieve this function?
![image](https://github.com/xcompact3d/Incompact3d/assets/103632822/5a72fa3a-7957-4fec-8f40-f0c689c524d1)
best,
Wwl

**Comment by slaizet** [comment, marked as answer]
Very hard to help you here as you are not providing any information about the reference case (2d/2d, Reynolds, averaged data or not....). Anyway, you can use Paraview to produce the same plots as in the reference data.
Best,
Sylvain
